config/application.rb

```diff
@@ -33,5 +35,15 @@ class Application < Rails::Application
     # Middleware like session, flash, cookies can be added back manually.
     # Skip views, helpers and assets when generating a new resource.
     config.api_only = true
+    config.middleware.insert_before 0, Rack::Cors do
+      allow do
+        origins 'http://your.frontend.domain.com'
```

Comment:
origins value does no seem right, ideally, this should be defined for specific env ie local dev, staging and production, I would strongly suggest we move this to env specific files